packages/block-library/src/footnotes/format.js

```diff
@@ -53,9 +52,11 @@ export const format = {
 			getBlockRootClientId,
 			getBlockName,
 			getBlockParentsByBlockName,
-		} = useSelect( blockEditorStore );
-		const footnotesBlockType = useSelect( ( select ) =>
-			select( blocksStore ).getBlockType( name )
+		} = registry.select( blockEditorStore );
+		const hasFootnotesBlockType = useSelect(
+			( select ) =>
+				!! select( blocksStore ).getBlockType( 'core/footnotes' ),
+			[]
```

Comment:
Here the code cares only whether the block is registered or not. Return a boolean from `useSelect` should be slightly faster.